### ISSUE TITLE
Support for UltraWide resolution (2560x1080)

### ISSUE THREAD

**Issue by sgtpep** [cubieplayer/Cubian#404]
I've tried Cubian Desktop HDMI version on Cubietruck and it sets resolution to 1920x1080. Also tried to force resolution by editing the `/boot/uEnv.txt`. Is it possible to get UltraWide resolution on Cubieboards?

**Comment by cubieplayer** [comment]
No, the resolution is fixed by 5 modes, see http://linux-sunxi.org/Fex_Guide#disp_init_configuration
